server/repository/src/db_diesel/barcode_row.rs

```diff
@@ -125,62 +112,3 @@ impl Upsert for BarcodeRow {
         )
     }
 }
-
-#[cfg(test)]
```

Comment:
I've removed these tests currently. Do you think we need to re-implement a similar pattern for every `changelog` table?